specs-go/config.go

```diff
@@ -22,11 +22,36 @@ type Spec struct {
 	Hooks Hooks `json:"hooks"`
 	// Annotations is an unstructured key value map that may be set by external tools to store and retrieve arbitrary metadata.
 	Annotations map[string]string `json:"annotations,omitempty"`
-
+	// HypervisorPath is the absolute path to the hypervisor binary
+	// for VM-based runtimes.
+	HypervisorPath string `json:"hypervisorPath"`
```

Comment:
On Mon, May 02, 2016 at 11:33:45AM -0700, Mrunal Patel wrote:
> > +	// HypervisorPath is the absolute path to the hypervisor binary
> > +	// for VM-based runtimes.
> > +	HypervisorPath string `json:"hypervisorPath"`
> 
> These settings should be optional and probably don't belong in the top level struct. 

Re: optional, see [1,2].  Re: top level, see [3] and later in that
thread.

[1]: https://github.com/opencontainers/runtime-spec/pull/405/files/6194f0b24749bedaf3f384787b73cf1c9f899f4b#r61262329
[2]: https://github.com/opencontainers/runtime-spec/pull/405/files/6194f0b24749bedaf3f384787b73cf1c9f899f4b#r61473810
[3]: https://groups.google.com/a/opencontainers.org/d/msg/dev/mM_DWZmXst0/uDPFv8-MBQAJ
     Subject: Re: Make the container spec unified for both native and virtual machine
     Date: Mon, 22 Feb 2016 15:12:46 -0800
     Message-ID: <20160222231246.GV4265@odin.tremily.us>